src/tools/wasm-ctor-eval.cpp

```diff
@@ -633,7 +632,7 @@ EvalCtorOutcome evalCtor(EvallingModuleRunner& instance,
   Literals params;
   for (Index i = 0; i < func->getNumParams(); i++) {
     auto type = func->getLocalType(i);
-    if (!LiteralUtils::canMakeZero(type)) {
+    if (LiteralUtils::canMakeZero(type)) {
```

Comment:
This looks wrong?

Reply:
Oops! A bug from restoring `canMakeZero`.